tests/EvmAcceptanceTests/hardhat.config.ts

```diff
@@ -27,6 +27,16 @@ const config: HardhatUserConfig = {
   defaultNetwork: "isolated_server",
 
   networks: {
+    mauro: {
+      url: "https://api.devnet.zilliqa.com",
+      websocketUrl: "https://api.devnet.zilliqa.com",
+      accounts: ["7498BC4F05582DBA6A5A2C11B86F080C071138B83B05396634A50175BE250D77"],
```

Comment:
Probably not a good idea to have live devnet keys in github - could we make this environment-variable configurable?

Reply:
Oh sorry. I accidentally merged that! I was helping Mauro.